python/ql/lib/semmle/python/web/flask/General.qll

```diff
@@ -8,7 +8,7 @@ deprecated ClassValue theFlaskClass() { result = Value::named("flask.Flask") }
 /** Gets the flask MethodView class */
 deprecated ClassValue theFlaskMethodViewClass() { result = Value::named("flask.views.MethodView") }
 
-deprecated ClassValue theFlaskReponseClass() { result = Value::named("flask.Response") }
+deprecated ClassValue theFlaskResponseClass() { result = Value::named("flask.Response") }
```

Comment:
The previous thing at least has a comment. I can't tell if this is deprecate because of the typo or just to show off deprecation...

Reply:
This one is deprecated for other reasons, and I think it would be better to _not_ fix the typo here (as we would have to keep the old version around for backwards compatibility anyway).